tests/files/config/test.json

```diff
@@ -0,0 +1,105 @@
+{
+    "prepare": {
+        "metadata": {
+            "label_field": "Compound_Concentration",
+            "label_composition": ["NCBIGeneID", "x_mutation_status"],
+            "control_field": "Compound_Concentration",
+            "control_value": "17"
+        },
+        "images": {
+            "channels": [
+                "R",
+                "G",
+                "B"
+              ],
+            "file_format": "tif",
+            "bits": 16
+        },
+        "illumination_correction": {
+            "down_scale_factor": 4,
+            "median_filter_size": 24
+        },
+        "compression": {
+            "implement": true,
+            "scaling_factor": 1.0
+        }
+    },
+    "train": {           
+        "gpus": "0",
+        "dset": {
+            "targets": [
+                "Class"
+            ],
+            "split_field": "Split",
+            "training_values": [0],
+            "validation_values": [1],
+            "width": 128,
+            "height": 128,
+            "mask_objects": false
+        },
+        "model": {
+            "name": "vae",
```

Comment:
what should we use for our default test model?

Reply:
For testing, I think the CNN model is the best. It's supervised, so it implicitly checks correct metadata usage as well.